src/composables/tippingContracts.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  computed, ref, shallowRef, Ref,
+} from '@vue/composition-api';
+import TIPPING_V1_INTERFACE from 'tipping-contract/Tipping_v1_Interface.aes';
+import TIPPING_V2_INTERFACE from 'tipping-contract/Tipping_v2_Interface.aes';
+import { Contract } from '@aeternity/aepp-sdk-13';
+
+import { useSdk13 } from './sdk13';
+import { watchUntilTruthy } from '../popup/utils';
+import {
+  IDefaultComposableOptions,
+  INetwork,
+  TippingV1ContractApi,
+  TippingV2ContractApi,
+} from '../types';
+
+const tippingV1 = shallowRef<Contract<TippingV1ContractApi> | null>(null);
+const tippingV2 = shallowRef<Contract<TippingV2ContractApi> | null>(null);
+const initializing = ref(false);
+
+export function useTippingContracts({ store }: IDefaultComposableOptions) {
+  const { getSdk } = useSdk13({ store });
+
+  const activeNetwork = computed<INetwork>(() => store.getters.activeNetwork);
+  const tippingSupported = computed(() => store.getters.tippingSupported);
+
+  async function initTippingContracts() {
+    if (!tippingSupported && !process.env.RUNNING_IN_TESTS) {
```

Comment:
I suggest to import `RUNNING_IN_TESTS` from the `environment.ts` file to have one typed source of truth.